enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/InternalAPIKeyAuthenticator.java

```diff
@@ -254,6 +261,42 @@ public AuthenticationContext authenticate(RequestContext requestContext) throws
                 APISecurityConstants.API_AUTH_GENERAL_ERROR, APISecurityConstants.API_AUTH_GENERAL_ERROR_MESSAGE);
     }
 
+    private void validateScopes(RequestContext requestContext, JWTClaimsSet payload, JWTTokenPayloadInfo jwtTokenPayloadInfo)
```

Comment:
let's discuss and decide if we need to set these scopes in the backend jwt token as well